jscomp/runtime/js_set.ml

```diff
@@ -9,12 +9,10 @@ external size : 'a t -> int = "size" [@@mel.get]
 external add : value:'a -> 'a t = "add" [@@mel.send.pipe: 'a t]
 external clear : 'a t -> unit = "clear" [@@mel.send]
 external delete : value:'a -> bool = "delete" [@@mel.send.pipe: 'a t]
-
-(* commented out until Melange has a plan for iterators
```

Comment:
there are some more of these comments throughout `jscomp/runtime` if you wanna tackle those too :)